### ISSUE TITLE
FK constraint fails on block and page deletion due to a missing cascade

### ISSUE THREAD

**Issue by ambroisemaupate** [BitBagCommerce/SyliusCmsPlugin#190]
```
An exception occurred while executing 'DELETE FROM bitbag_cms_block WHERE id = ?' with params [4]: SQLSTATE[23000]: Integrity constraint violation: 1451 Cannot delete or update a parent row: a foreign key constraint fails (`chni_eu`.`bitbag_cms_block_products`, CONSTRAINT `FK_C4B9089FC4663E4` FOREIGN KEY (`page_id`) REFERENCES `bitbag_cms_block` (`id`))
```

Steps to reproduce:

- Create a *Sylius* product
- Create a *Bitbag* page
- Link product into your new page
- Delete that page… 

I suspect this to be due to ORM mapping which misses a `onDelete: CASCADE` on `joinColumn`:

https://github.com/BitBagCommerce/SyliusCmsPlugin/blob/master/src/Resources/config/doctrine/Block.orm.yml#L26

You should add `onDelete: CASCADE` on `joinColumn`, *Doctrine* does not add them by default on joinColumn **and** inverseJoinColumn if you are changing columns configuration. Here, it’s the missing `CASCADE` on joinColumn which is faulty and prevent me to delete a block or a page since they’re linked to a product.

**Comment by nicoplh** [comment]
Hi, 

Is it possible to merge those changes ? :)

**Comment by mikemix** [comment]
Same here, cannot delete blocks.

**Comment by bitbager** [comment]
Can I ask you guys to provide a PR?

**Comment by Holicz** [comment]
Would it be possible to merge this? :)

**Comment by aayaresko** [comment]
new pr - #236, build passed

**Comment by patrick477** [comment]
Fixed #232
